can-control.js

```diff
@@ -282,11 +282,15 @@ var Control = Construct.extend(
 				pluginname = cls.pluginName || cls.shortName,
 				arr;
 
-			// Retrieve the raw element, then set the plugin name as a class there.
+			if(!element) {
+				dev.warn('can/control/control.js: Creating an instance of a named control without passing an element');
 
+				element = document.createElement('div');
```

Comment:
do you need to create an element if you're also checking that the element exists below?